README.md

```diff
@@ -1,5 +1,6 @@
 [![Build Status](https://travis-ci.org/pardom/ActiveAndroid.png?branch=master)](https://travis-ci.org/pardom/ActiveAndroid) [![Stories in Ready](https://badge.waffle.io/pardom/ActiveAndroid.png)](http://waffle.io/pardom/ActiveAndroid)  
-# ActiveAndroid
+# ActiveAndroid [![](https://jitpack.io/v/aikongmeng/ActiveAndroid.svg)](https://jitpack.io/#aikongmeng/ActiveAndroid)
```

Comment:
why `aikonmeng`? this repository belongs to @pardom-zz...

Reply:
I thought the author was not maintaining the project。
Replace this with `aikongmeng` ,  just for myself use.
Sorry,I wanted to Pull ,Push was Error operation.

Reply:
> why `aikonmeng`? this repository belongs to @pardom-zz...

I thought the author was not maintaining the project。
Replace this with aikongmeng , just for myself use.
Sorry,I wanted to Pull ,Push was Error operation.